src/lib/profiles/data-management/Current/LoggingManagement.cpp

```diff
@@ -1094,6 +1094,10 @@ event_id_t LoggingManagement::LogEvent(const EventSchema & inSchema, EventWriter
 
 exit:
     Platform::CriticalSectionExit();
+    if (event_id == 0)
+    {
+        WeaveLogProgress(EventLogging, "Failed to log event, return event id 0");
```

Comment:
Recommend logging the ProfileId from the inSchema as well.

Reply:
looks resolved.